typescript/test/util.ts

```diff
@@ -0,0 +1,104 @@
+import * as assert from 'assert';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+
+const BASELINE_EXTENSION = '.baseline';
+export function comapreToBaseline(outpurDir: string, baselineDir: string) {
```

Comment:
`compareToBaseline` (typo, it says `comapre`)

Reply:
done.